### PR TITLE
[ADD] Change font color of  calendar event from  task category form

### PR DESCRIPTION
**Pull request by jananjoy**

### CODE REVIEW

File: project_event/models/calendar_event.py [hunk 1]
```diff
@@ -21,6 +21,10 @@ class CalendarEvent(models.Model):
         string='Client Type',
     )
     color = fields.Char(related='category_id.color')
+    font_color = fields.Selection([
+        ('black', 'Black(Default)'),
```
Comment (eilst) [inline project_event/models/calendar_event.py]: 'Black(Default)' => 'Black (Default)'

File: project_event/models/calendar_event.py [hunk 2]
```diff
@@ -21,6 +21,10 @@ class CalendarEvent(models.Model):
...
+        ('white', 'White')],
```
Comment (eilst) [inline project_event/models/calendar_event.py:26]: Add translations

File: project_event/views/project_task_view.xml [hunk 3]
```diff
@@ -196,6 +196,7 @@
                                    domain="[('activity_task_type','=','activity')]" attrs="{'readonly': [('task_state', 'in', ['done','accepted'])]}"/>
                             <field name="category_id" required="1"  attrs="{'readonly': [('task_state', 'in', ['done','accepted'])]}"/>
                             <field name="color" widget="color" readonly="1" attrs="{'readonly': [('task_state', 'in', ['done','accepted'])]}"/>
+                            <field name="font_color" widget="radio" readonly="1" attrs="{'readonly': [('task_state', 'in', ['done','accepted'])]}"/>
```
Comment (rimbendhaou) [inline project_event/views/project_task_view.xml]: no need to display this field, put it invisible. remove readonly=1 and attrs=readonly

File: project_event/models/calendar_event.py (hunk 1 above)
Comment (rimbendhaou) [reply project_event/models/calendar_event.py]: add space between Black and (

**Review by rimbendhaou — CHANGES_REQUESTED** [review]

File: project_event/models/calendar_event.py (hunk 1 above)
Comment (jananjoy) [reply project_event/models/calendar_event.py]: @rimbendhaou  ok

File: project_event/views/project_task_view.xml (hunk 3 above)
Comment (jananjoy) [reply project_event/views/project_task_view.xml]: @rimbendhaou  ok

File: project_event/models/calendar_event.py (hunk 2 above)
Comment (jananjoy) [reply project_event/models/calendar_event.py:26]: @rimbendhaou  ok

**Review by rimbendhaou — APPROVED** [review]

**Pull request merged**